pages/docs/reference/java-interop.md

```diff
@@ -752,9 +752,6 @@ executor.execute(Runnable { println("This runs in a thread pool") })
 Note that SAM conversions only work for interfaces, not for abstract classes, even if those also have just a single
 abstract method.
 
-Also note that this feature works only for Java interop; since Kotlin has proper function types, automatic conversion
-of functions into implementations of Kotlin interfaces is unnecessary and therefore unsupported.
```

Comment:
Let's provide a link to `fun` interface here and say something like "Starting from version 1.4, it's possible to use `fun` interace for SAM conversions..."

Reply:
Added cross-links between articles. Thx! I forgot 🙈
But as I described above, I don't use versions except for What's new. 🙂

Reply:
BTW, in docs we use "you" according to the JB style guide (Google and MS style guides as well). It's better to avoid "it's possible, one can". We refer to the reader. This makes docs more personal and friendly. 🤗